ietf-gss-kex-qr.xml

```diff
@@ -102,7 +102,9 @@
     <section title="New Quantum Resistant Key Exchange Methods">
       <t>This document adopts the same naming convention as defined in <xref
         target="RFC4462"/> to define families of methods that cover any
-        GSS-API mechanism used with a specific SHA-2 Hash.
+          GSS-API mechanism used with a specific SHA-2 Hash. It also
+          reuses much of the the scheme defined in Section 2.1 of
```

Comment:
Does 'reuses most of the scheme' sound better? Not sure :)

Reply:
yeah, but "most" implies majority, and I don't think we're there :)